client/src/App.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import { Route, BrowserRouter as Router } from 'react-router-dom';
 import GetDetails from './components/GetDetails';
+import Upload from './components/Upload';
 
 function App() {
   return (
     <Router>
-      <Route path="/GetDetails" component={GetDetails} />
+      <div className="App">
+        <Route path="/GetDetails" component={GetDetails} />
```

Comment:
You need to keep your path's consistent. They should all be kebab case, like `upload-photo`.